src/_guides/language/language-tour.md

```diff
@@ -7,6 +7,13 @@ canonical: https://dart.dev/language
 noindex: true
 ---
 
+{{site.alert.warning}}
+  The Dart language tour's content has been split across
+  several different pages in the sidenav under **Language**.
```

Comment:
**issue:** `sidenav` is jargon and doesn't translate. Consider:
```suggestion
  several different pages in the left side navigation under **Language**.
```

Reply:
Thanks, I've made the change!